docs/content/how-tos/rules/FL0087.md

```diff
@@ -0,0 +1,34 @@
+---
+title: FL0087
+category: how-to
+hide_menu: true
+---
+
+# NoAsyncRunSynchronouslyInLibrary (FL0087)
+
+*Introduced in `0.26.7`*
+
+## Cause
+
+`Async.RunSynchronously` method is used to run async computation in library code.
+
+The rule assumes the code is in the library if none of the following is true:
+- The code is inside NUnit or MSTest test.
+- Namespace or project name contains "test" or "console".
+- Assembly has `[<EntryPoint>]` attribute one one of the functions/methods.
+
+## Rationale
+
+Using `Async.RunSynchronously` outside of scripts and tests can lead to program becoming non-responsive.
+
+## How To Fix
+
+Remove `Async.RunSynchronously` and wrap the code that uses `async` computations in `async` computation, using `let!`, `use!`, `match!`, or `return!` keyword to get the result.
```

Comment:
@webwarrior-ws this can be clearer, for example by also mentioning that if the method/function returned type TFoo, then the new version might return `Async<TFoo>` instead, in which case it might be better to prefix its name with `Async` (and in case the method/function is public, a nice C#-friendly overload that returns `Task<TFoo>` could be provided, suffixed with `Async`, that just calls the previous method with Async.StartAsTask. And to be able to explain all this, maybe it's better to attach code samples (there's some rules that have multi-line code samples already, have a look at those to follow the same format).